src/flap.sol

```diff
@@ -112,19 +115,19 @@ contract Flapper is LibNote {
         emit Kick(id, lot, bid);
     }
     function tick(uint id) external note {
-        require(bids[id].end < now);
-        require(bids[id].tic == 0);
+        require(bids[id].end < now, "Flapper/not-finished");
+        require(bids[id].tic == 0, "Flapper/already-bidded");
         bids[id].end = add(uint48(now), tau);
     }
     function tend(uint id, uint lot, uint bid) external note {
-        require(live == 1);
-        require(bids[id].guy != address(0));
-        require(bids[id].tic > now || bids[id].tic == 0);
-        require(bids[id].end > now);
+        require(live == 1, "Flapper/not-live");
+        require(bids[id].guy != address(0), "Flapper/not-set");
```

Comment:
"Flapper/guy-not-set"?

Reply:
This is actually checking that the auction is live/initiated, i.e. that you're not bidding on a `0` auction. So maybe
`require(live == 1, "Flapper/flapper-not-live");`
and 
`require(bids[id].guy != address(0), "Flapper/auction-not-live");`